src/collector/mod.rs

```diff
@@ -196,6 +196,7 @@ pub fn collect_local_repo(repo_path: &Path) -> Result<RepoResult> {
     Ok(RepoResult { meta, metrics })
 }
 
+#[instrument(skip(files))]
```

Comment:
Pls remove, I've already added spans inside w/o the `instrument` (using `info_span!`)